ibl_pipeline/analyses/load_mouse_data_datajoint.py

```diff
@@ -0,0 +1,142 @@
+﻿# from oneibl.one import ONE
+import pandas as pd
+import numpy as np
+from os import listdir, getcwd
+from os.path import isfile, join
+import re
+from IPython import embed as shell
+
+import datajoint as dj
+from ibl_pipeline import reference, subject, action, acquisition, data, behavior
+
+
+# ==================== #
+# DATAJOINT
+# ==================== #
+
+def get_weights(mousename):
+    mouse = (subject.Subject() & 'subject_nickname = "%s"' % mousename)
+
+    wei = {}
+    wei['date_time'], wei['weight'] = (action.Weighing() & mouse).fetch('weighing_time',
+                                                                        'weight', order_by='weighing_time')
```

Comment:
same here. mouse is no longer necessary. `action.Weighing & 'subject_nickname=mouse'` would work. By the way, brackets after `action.Weighing` are also not necessary.